ui/qtgalacticplot.py

```diff
@@ -31,9 +31,13 @@ def __init__(self, parent: QWidget = None):
         self.__annotate.set_visible(False)
         self.__planetNames = []
         self.__planetOwners = []
+        self.__starbaseLevel = []
+        self.__shipyardLevel = []
+        self.__income = []
+        self.__groundStructureSlots =[]
         self.__planetsScatter = None
 
-    def plotGalaxy(self, planets, tradeRoutes, allPlanets, planetOwners = [], autoPlanetConnectionDistance: int = 0) -> None:
+    def plotGalaxy(self, planets, tradeRoutes, allPlanets, planetOwners = [], starbaseLevel = [], shipyardLevel = [], groundStructureSlots = [], income = [], autoPlanetConnectionDistance: int = 0) -> None:
```

Comment:
I don't think this change to `plotGalaxy` is needed here: the variables aren't being populated this way. 

The variables are created at lines 34-37 above and then the values come in with allPlanets (properties of the Planet class) and are appended into the variables at lines 80-83 below.